pkg/types/clustermetadata.go

```diff
@@ -9,8 +9,12 @@ import (
 // ClusterMetadata contains information
 // regarding the cluster that was created by installer.
 type ClusterMetadata struct {
-	ClusterName             string `json:"clusterName"`
-	ClusterID               string `json:"clusterID"`
+	// clusterName is the name for the cluster.
+	ClusterName string `json:"clusterName"`
+	// clusterID is a globally unique ID that is used to identify an Openshift cluster.
+	ClusterID string `json:"clusterID"`
+	// infraID is an ID that is used to identify cloud resources created by the installer.
+	InfraID                 string `json:"infraID"`
```

Comment:
Now that we are using a different term for the human-readable cluster ID somewhere, is it worth revisiting whether it would be better to use that term everywhere the human-readable cluster ID is used?

Reply:
`metadata.json` is an output to the users like hive, they don't care what installer used as ID, just that its the ID used to tag/name resources. 
`Human readable ID` IMO is an installer's internal impl and doesn't need to bleed into metadata.json

Reply:
I updated the `ClusterID.HumanID` to `ClusterID.InfraID`

/cc @crawford @staebler